Team Pattern Created

devmuhib009 opened the pull request:
Updated Team Pattern

**Description**

Just created and updated team pattern

devmuhib009 committed:
Team Pattern Created

Updated Team Pattern

devmuhib009 committed:
Team Pattern PHP Validation

Updated PHP Validation

maneshtimilsina reviewed (commented):
It will be better to remove the ID and class of the images.

carolinan commented:
This PR replaces https://github.com/WordPress/twentytwentyfour/pull/24

huzaifaalmesbah commented:
Related #117

carolinan commented:
We need to include the original images without the rounded corners and only change the radius with CSS.

devmuhib009 commented:
When I save images from Figma file, it comes with rounded corner. However, I will recheck and add original images with CSS code to make them rounded.

carolinan commented:
To clarify I did not mean adding any CSS to the theme. I think the rounded style should be used, like in this PR.

devmuhib009 committed:
Team Member Block Updated

ID and Class Removed from Images.

devmuhib009 committed:
Merge branch 'WordPress:trunk' into pattern/team.php

MaggieCabrera commented on patterns/team.php:
```diff
@@ -0,0 +1,85 @@
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","scale":"cover","sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"16px"}}} -->
+<figure class="wp-block-image aligncenter size-full has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team-member-3.png" alt="" style="border-radius:16px;object-fit:cover"/></figure>
```
This image already exists in the assets folder named avatar_4.jpg and it's used in the sidebar pattern. I'm happy with renaming it there if you want, but let's not have duplicated assets in any case

devmuhib009 replied:
Sure. You can remove the duplicate one.

MaggieCabrera commented on patterns/team.php:
```diff
@@ -0,0 +1,85 @@
+/**
+ * Title: Team
+ * Slug: twentytwentyfour/team
+ * Categories: team
```
This category doesn't exist, let's use featured and text instead

devmuhib009 replied:
Ok

MaggieCabrera commented on patterns/team.php:
```diff
@@ -0,0 +1,85 @@
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:image {"align":"center","scale":"cover","sizeSlug":"full","linkDestination":"none","style":{"border":{"radius":"16px"}}} -->
+<figure class="wp-block-image aligncenter size-full has-custom-border"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/team-member-2.png" alt="" style="border-radius:16px;object-fit:cover"/></figure>
```
let's make use of aspect ratio here and use the square one so all images are the same size

devmuhib009 replied:
Ok

MaggieCabrera reviewed (changes requested):
This is generally looking great, just a few comments and it should be good to go

devmuhib009 commented:
@MaggieCabrera I will update the changes and inform you.

maneshtimilsina reviewed (changes requested):
- Space/padding at the top and bottom of the section is missing. Please check Figma and update it accordingly. Let's use the spacing presets for paddings where needed.
- Make it align full instead of wide
- Font size of the Subheader is larger than it is set in Figma
- You are using margin-top 64px for the row used in the pattern. Instead of arbitrary values, please use the spacing presets.
- I would suggest using columns instead of row to display team members in a grid.
- Replace current images with large size images and check once.

devmuhib009 committed:
Team Pattern Changed with Category Modification

I just modified everything following comments I got after my last commit.

devmuhib009 committed:
Merge branch 'WordPress:trunk' into pattern/team.php

MaggieCabrera commented on patterns/team.php:
```diff
@@ -2,84 +2,91 @@
 
 <!-- wp:paragraph {"align":"center","fontSize":"small"} -->
-<p class="has-text-align-center has-small-font-size"><?php echo esc_html_x( 'Founder, CEO & Architect', 'Team Member Designation', 'twentytwentyfour' ); ?></p>
+<p class="has-text-align-center has-small-font-size"><?php echo esc_html__( 'Founder, CEO &amp; Architect', 'twentytwentyfour' ); ?></p>
```
why did you remove the description of the translations?

devmuhib009 replied:
I did some structure change, so previous translation and other things removed automatically. So, I did all manually again. In most of the files I noticed there is esc_html__ being used. So, I thought both are ok.

devmuhib009 committed:
Translation Version Updated

Changed esc_html__ to esc_html_x

huzaifaalmesbah commented:
Looks Good

devmuhib009 commented:
Thank you :)

devmuhib009 committed:
Restored Previous Avatar_4 Image

I accidently deleted this with new team-member image. Now I restored it from recycle bin.

devmuhib009 committed:
Image Height Issue Fixed

After restoring image Avatar_4.jpg that is being used in sidebar.php is larger than originial team images. So, I added height to adjust layout.

huzaifaalmesbah commented:
Add image alt text and ready for translation `esc_attr__`

devmuhib009 commented:
Can you please give an example or where I can find this? Please give a file name where I will find sample of alt. Thanks

huzaifaalmesbah commented:
you can check for example [esc_attr__](https://github.com/WordPress/twentytwentyfour/pull/192/commits/4109d5745bfae8f1ed55f877e25c41d9abf0da2e)

devmuhib009 commented:
ok

devmuhib009 committed:
Alt Tag Added

devmuhib009 commented:
> you can check for example [esc_attr__](https://github.com/WordPress/twentytwentyfour/pull/192/commits/4109d5745bfae8f1ed55f877e25c41d9abf0da2e)

Added the alt tag. Thanks for the suggestion.

huzaifaalmesbah reviewed (commented):
Looks Good

MaggieCabrera committed:
renamed image, tweaked markup and image descriptions

MaggieCabrera reviewed (approved):
I made a couple of changes, this is good to go, thank you for your work!

devmuhib009 commented:
Thank you! Learned lots of things from this specific contribution. ❤️❤️❤️
